src/components/layout/NavBar.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+ <div>
```

Comment:
You might want to configure your editor to "format on save" - and use 2 spaces for vue files ...

Otherwise formatting might get out of hand very quickly (i'll make sure to also document the settings!)

Reply:
I'll look into how to do that. I'm using vscode for this. Normally, I use pycharm CE.

Reply:
I'll send you the config i use for it - having vscode "autoformat" on save is quite ... handy ;)

Reply:
I am using the prettier vscode extension which does this for me already, but I think it's acting strange because of the additional eslint and prettier configs and my limited javascript linting experience. A nudge in the right direction would help :)

Reply:
So it's here for future reference:
i'm using the vscode eslint extension - with the following setting.
```
   "editor.codeActionsOnSave": {
        "source.fixAll.eslint": true
    }
```

This formats the code each time i save ... (side note: Format Document does currently not work for whatever reason - but i don't really need it as save does the same thing.